www/src/data/sidebars/doc-links.yaml

```diff
@@ -469,6 +469,8 @@
                   link: /docs/processing-payments-with-stripe/
                 - title: Processing Payments with Square
                   link: /docs/processing-payments-with-square/
+                - title: Guide on working with Etsy in Gatsby
```

Comment:
```suggestion
                - title: Working with Etsy in Gatsby
```

Reply:
I am marking this as unresolved, as it needs to be adjusted as part of this PR.